src/tree/v2/azure/GroupingItem.ts

```diff
@@ -29,6 +30,7 @@ export class GroupingItem implements ResourceGroupsItem {
         public readonly context: ResourceGroupsTreeContext,
         private readonly resourceItemFactory: ResourceItemFactory<AzureResource>,
         private readonly branchDataProviderFactory: (azureResource: AzureResource) => AzureResourceBranchDataProvider<AzureResourceModel>,
+        private readonly onChangeBranchDataProviders: vscode.Event<AzExtResourceType>,
```

Comment:
Can we change the name to match VSCode's naming conventions for events?
```suggestion
        private readonly onDidChangeBranchDataProviders: vscode.Event<AzExtResourceType>,
```